parmed/openmm/parameters.py

```diff
@@ -667,7 +678,7 @@ def _get_atoms_with_external_bonds(self, residue_name):
                 element_name = atom.element_name
 
             # If the number of bonds within the residue does not equal the number of bonds the atom should have, the atom  has an external bond 
```

Comment:
This comment looks out of date now?  Why this change?

Reply:
Yes, it's out of date. I changed this because it was a mistake in my original code -- we are assuming that any extra bonds are external bonds. If there aren't extra bonds, there are no external bonds.